tests/utils.go

```diff
@@ -4330,7 +4330,11 @@ func resetToDefaultConfig() {
 	}
 
 	UpdateKubeVirtConfigValueAndWait(KubeVirtDefaultConfig)
-	UpdateCDIConfigMap(CDIInsecureRegistryConfig)
+
+	// On SRIOV we dont have Insecure registries
```

Comment:
I think we do not have CDI at all. I do not know what is this insecure registries.

@ormergi ?

Reply:
insecure registries are registries that you dont need to validate their access,
i will post in a bit the PR that removed them.

Reply:
We do not deploy cdi-provider on sriov lane (due to https://github.com/kubevirt/kubevirt/issues/4120)

Every kubevirtci provider creates local images registry container where we push to it all kubevirt components.
Since its just for testing/development we do not sign it, thus so called "insecure registry"

Actually #4419 is for cancel the checking if that config-map exists